### ISSUE TITLE
Is or would there be(e) a way to have the honey walls be non-traversable?

### ISSUE THREAD

**Issue by Niniisan** [Jack7D1/Rimworld-Apini#132]
I love, LOVE that mod and I'm making a save using Biotech, of wasps people using the Apinis as slaves. Anyway, I'm using the honey walls to make a hive, but they're traversable and break all the rooms' temperatures and  sizes. Would there be any way to make them non-traversable, or maybe another version of that wall that's "harder" like wax without having to use the base walls and just change the material? Thank you!

**Comment by Jack7D1** [comment]
Hey!
I've been thinking about this for a while.
Honey walls are treated similarly to furniture, as such they can't do much more than slow pawns down. That was the original intent but It doesn't seem that honey walls are used much at all.
Might be worth cutting them like honey floors.
What you seem to be wanting is a wall with honey material, this could be used as a beauty wall like silver/gold walls in my opinion

**Comment by Niniisan** [comment]
Yes! Obviously with how the crafting the honey walls currently works, I do think treating a wall with honey material like a beauty wall would work, as long as they're non-traversable
